### PR TITLE
Add issue template (issue #942)

### LINKED ISSUE #942: Add issue templates
as per
https://github.com/fnielsen/scholia/community .

Some good examples in https://github.com/ytdl-org/youtube-dl/tree/master/.github .

### PR DESCRIPTION
**Pull request by DanielFLopez**
fix #942

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Community Support
+    url: https://github.community/
+    about: Please ask and answer questions here.
+  - name: GitHub Security Bug Bounty
+    url: https://bounty.github.com/
+    about: Please report security vulnerabilities here.
```
Comment (egonw) [inline .github/ISSUE_TEMPLATE/config.yml:8]: Is this content correct? should that not point to Scholia?

File: .github/ISSUE_TEMPLATE/_bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+---
+name: Bug report
+about: Report a bug unrelated to any particular site or extractor
+label: check relevant labels to the issue
```
Comment (egonw) [inline .github/ISSUE_TEMPLATE/_bug_report.md:5]: there is a `bug` label too

File: .github/ISSUE_TEMPLATE/_bug_report.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+[ordered list the process to finding and recreating the issue, example below]
+
+1. go to this url https://scholia.toolforge.org/author/
+2. click the random author button
+3. the error is raised
+
+## Expected result
+
+[describe what you would expect to have resulted from this process]
```
Comment (egonw) [inline .github/ISSUE_TEMPLATE/_bug_report.md:24]: Suggestion: add the suggestion to people to upload screenshots here too

File: .github/ISSUE_TEMPLATE/_feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
+---
+name: Feature request
```
Comment (egonw) [inline .github/ISSUE_TEMPLATE/_feature_request.md:3]: I can imagine we want two special kind of feature requests (what do you think? this is not blocking this PR):

1. a request to add a new `aspect` (like we have `work`, `protein`, etc)
2. a request to add a new panel, where people can post their SPARQL they suggest to be used

**Review by egonw — COMMENTED** [review]
Some comments/ideas/question. Nothing really blocking this, tho.

File: .github/ISSUE_TEMPLATE/_feature_request.md (hunk 4 above)
Comment (DanielFLopez) [reply .github/ISSUE_TEMPLATE/_feature_request.md:3]: yes, then should I split this template for both cases?

File: .github/ISSUE_TEMPLATE/_bug_report.md (hunk 2 above)
Comment (DanielFLopez) [reply .github/ISSUE_TEMPLATE/_bug_report.md]: done

File: .github/ISSUE_TEMPLATE/_bug_report.md (hunk 3 above)
Comment (DanielFLopez) [reply .github/ISSUE_TEMPLATE/_bug_report.md]: done

File: .github/ISSUE_TEMPLATE/_bug_report.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+[provide general introduction to the issue logging and why it is relevant to this repository]
+
+## Context
+
+[provide more detailed introduction to the issue itself and why it is relevant]
+
+## Process
+
+[ordered list the process to finding and recreating the issue, example below]
```
Comment (Daniel-Mietchen) [inline .github/ISSUE_TEMPLATE/_bug_report.md:16]: This should be 
```[ordered list of the process to find and reproduce the issue, example below]```

File: .github/ISSUE_TEMPLATE/_bug_report.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+2. click the random author button
+3. the error is raised
+
+## Expected result
+
+[describe what you would expect to have resulted from this process, include a screenshot if possible]
+
+## Current result
+
+[describe what you you currently experience from this process, and thereby explain the bug]
```
Comment (Daniel-Mietchen) [inline .github/ISSUE_TEMPLATE/_bug_report.md:28]: This should be
```[describe what you currently experience from this process, and thereby explain the bug]```

File: .github/ISSUE_TEMPLATE/_feature_request.md (hunk 4 above)
Comment (Daniel-Mietchen) [reply .github/ISSUE_TEMPLATE/_feature_request.md:3]: Yes, something like 
name: Feature request, e.g.

* request to add a new `aspect` (in addition to the existing ones like `work`, `protein` etc.)
* request to add a new `panel` within an existing `aspect`, ideally with a suggestion for a suitable SPARQL query

File: .github/ISSUE_TEMPLATE/_feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+
+## Context
+
+[provide more detailed introduction to the feature itself and why it is relevant]
+
+## Process
+
+[ordered list process for new functionality (optional)]
```
Comment (Daniel-Mietchen) [inline .github/ISSUE_TEMPLATE/_feature_request.md:15]: This should be 
```[ordered list of process for new functionality (optional)]```

**Review by Daniel-Mietchen — COMMENTED** [review]
Needs some minimal text changes.

**Pull request merged**